yt_dlp/extractor/nrk.py

```diff
@@ -163,33 +171,36 @@ def call_playback_api(item, query=None):
         playable = manifest['playable']
 
         formats = []
+        subtitles = {}
         for asset in playable['assets']:
             if not isinstance(asset, dict):
                 continue
             if asset.get('encrypted'):
-                continue
+                pass  # Unencrypted stream no longer available
```

Comment:
If I recall correctly I was testing with the NRK1 live feed manifest when I added that comment, which no longer returned any unencrypted streams. Here's the assets array it returns with default headers/parameters for the web player:
`https://psapi.nrk.no/playback/manifest/channel/nrk1?eea-portability=true`
```json
"assets": [
            {
                "url": "https://nrk-nrk1-ostlandet.akamaized.net/21/0/hls/nrk_1/playlist.m3u8?bw_low=231&bw_high=12563&bw_start=1228&no_iframes&no_audio_only",
                "format": "HLS",
                "mimeType": "application/vnd.apple.mpegurl",
                "encrypted": true,
                "encryptionScheme": "statickey"
            }
        ],
```

Snippet from the 1080p stream inside playlist.m3u8:
```
#EXTM3U
#EXT-X-VERSION:3
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:30904434
#EXT-X-PROGRAM-DATE-TIME:2024-10-16T21:39:01.792Z
#EXT-X-KEY:METHOD=AES-128,URI="keys/1.key",IV=0x865832653759693f5f5ad4df269caaad
#EXTINF:10.000000,
1035432803-1026273474-27813990761347.ts?version_hash=876370db
[...]
```
 I just made the assumption that this was the case for all streams now, but apparently not since programs still return an unencrypted stream:
`https://psapi.nrk.no/playback/manifest/program/MUHH63000123?eea-portability=true`
```json
 "assets": [
            {
                "url": "https://nrkod77-cdn0-47115-odedge1.dna.contentdelivery.net/47115-odedge1/open/ps/muhh/muhh63000123/ac7a7cb3-23.smil/cmaf.m3u8?adap=large",
                "format": "HLS",
                "mimeType": "application/vnd.apple.mpegurl",
                "encrypted": false,
                "encryptionScheme": "none"
            }
        ],
```

I'm not sure what the `statickey` encryption scheme is, but the m3u8 manifest still played in VLC without issues, so I just changed the `continue` statement to a `pass` and added the comment.

Reply:
> #EXT-X-KEY:METHOD=AES-128,URI="keys/1.key",IV=0x865832653759693f5f5ad4df269caaad

This looks like it's just HLS AES-128 envelope encryption; not DRM. It can be handled by yt-dlp's native HLS downloader as well as ffmpeg

Reply:
Okay with these being clearkey you can probably drop the entire `if asset.get('encrypted'):` section instead of changing the `continue` to `pass`.

Reply:
Allright! I'll remove it. Though, it should be something to be aware of in the future if they start using actual DRM on specific content. It should maybe also use a more specific exception too at that point, about DRM not being supported, since just naively skipping streams hoping there is another one probably isn't the best approach.

Reply:
If you have any specific streams using actual DRM, we can add some logic here. Otherwise just continuing is fine. yt-dlp should detect DRM protected streams that can't be downloaded and raise an appropriate warning by itself.